test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.h

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _CL_KHR_BASIC_COMMAND_BUFFER_H
+#define _CL_KHR_BASIC_COMMAND_BUFFER_H
+
+#include "command_buffer_test_base.h"
+#include "harness/typeWrappers.h"
+
+// temporary flag to be removed once simultaneous test will be sorted out
+#define USE_COMMAND_BUF_KENEL_ARG 0
```

Comment:
A note that I think the approach with this flag set is incorrect, but it passed on one implementation, so the code is in this draft PR so that we can reference it on the WG teleconference

Reply:
Correct, just for completeness, setting this flag to 1 enables the path which calls clSetKernelArg between queuing simultaneous command buffers which according to specification is not correct approach but makes the test scenario work with POCL library.

Reply:
Mentioned flag was removed due to narrowed scope of test scenario which works now with both POCL and Command Buffer Emulation Layer.